source/scripting_v3/GTA/Entities/Vehicles/Vehicle.cs

```diff
@@ -55,23 +55,84 @@ public void Explode()
+        /// </summary>
         public bool HasBulletProofGlass => SHVDN.NativeMemory.HasVehicleFlag(Model.Hash, SHVDN.NativeMemory.VehicleFlag2.HasBulletProofGlass);
+
+        /// <summary>
+        /// Gets a value indicating whether this <see cref="Vehicle"/> has Lowrider Hydraulics.
+        /// </summary>
+        /// <remarks>
+        /// Returns <see langword="false"/> prior to <c>v1.0.505.2</c>;
+        /// </remarks>
         public bool HasLowriderHydraulics
             => Game.FileVersion >= VersionConstsForGameVersion.v1_0_505_2
             && SHVDN.NativeMemory.HasVehicleFlag(Model.Hash, SHVDN.NativeMemory.VehicleFlag2.HasLowriderHydraulics);
+
+        /// <summary>
+        /// Gets a value indicating whether this <see cref="Vehicle"/> has Donk Hydraulics.
+        /// </summary>
+        /// <remarks>
+        /// Returns <see langword="false"/> prior to <c>v1.0.505.2</c>;
+        /// </remarks>
         public bool HasDonkHydraulics
             => Game.FileVersion >= VersionConstsForGameVersion.v1_0_505_2
             && SHVDN.NativeMemory.HasVehicleFlag(Model.Hash, SHVDN.NativeMemory.VehicleFlag2.HasLowriderDonkHydraulics);
+
+        /// <summary>
+        /// Gets a value indicating whether this <see cref="Vehicle"/> has a Parachute.
+        /// </summary>
+        /// <remarks>
+        /// Returns <see langword="false"/> prior to <c>v1.0.505.2</c>;
+        /// </remarks>
         public bool HasParachute
             => Game.FileVersion >= VersionConstsForGameVersion.v1_0_505_2
             && Function.Call<bool>(Hash.GET_VEHICLE_HAS_PARACHUTE, Handle);
+
+        /// <summary>
+        /// Gets a value indicating whether this <see cref="Vehicle"/> has a Rocket Boost
+        /// </summary>
+        /// <remarks>
+        /// Returns <see langword="false"/> prior to <c>v1.0.944.2</c>;
+        /// </remarks>
         public bool HasRocketBoost
             => Game.FileVersion >= VersionConstsForGameVersion.v1_0_944_2
             && Function.Call<bool>(Hash.GET_HAS_ROCKET_BOOST, Handle);
+
+        /// <summary>
+        /// Gets a value indicating whether this <see cref="Vehicle"/> has the Parachute deployed.
+        /// </summary>
+        /// <remarks>
+        /// Returns <see langword="false"/> prior to <c>v1.0.1011.1</c>;
+        /// </remarks>
         public bool IsParachuteDeployed
             => Game.FileVersion >= VersionConstsForGameVersion.v1_0_1011_1
             && Function.Call<bool>(Hash.IS_VEHICLE_PARACHUTE_DEPLOYED, Handle);
+
+        /// <summary>
+        /// Gets or sets a value indicating whether this <see cref="Vehicle"/> has its rocket boost currently active.
+        /// </summary>
+        /// <remarks>
+        /// Reading always <see langword="false"/> prior to <c>v1.0.1011.1</c>;
+        /// Writing to this prior to <c>v1.0.1011.1</c> will throw an error.
+        /// </remarks>
         public bool IsRocketBoostActive
```

Comment:
I think this doc is just missing what kind of exception can be thrown from the setter, but let's have it mention, which is `GameVersionNotSupportedException`.